### ISSUE TITLE
Welcome to the efficientteacher project

### ISSUE THREAD

**Issue by BowieHsu** [AlibabaResearch/efficientteacher#3]
请允许我介绍一下3月的更新计划——1.我们将会在验证后上传在readme中提到的这些coco预训练模型，这样可以进一步方便大家调试和使用我们的算法库，也能更快地获得较好的半监督训练效果；2.我们的半监督方案还无法适应任意的数据分布，所以我们希望收集大家在实际使用中遇到的各种情况，请随时提出issue来共同改进；3.针对中文用户，我们将会写更多的教程放在知乎平台上，针对英文用户，我们还没有找到一个好的渠道，有能提供帮助的朋友也请留下您的建议，非常感谢大家对项目的关注。

Let me introduce the update plan for March: 1. After verification, we will upload the coco pre-trained models mentioned in readme, which will further facilitate everyone to debug and use our algorithm library, and can also get better semi-supervised training effect faster; 2. Our semi-supervised scheme can not adapt to any data distribution, so we hope to collect various situations encountered by everyone in actual use, please feel free to submit issues to improve together; 3. For Chinese users, we will write more tutorials on the Zhihu platform. For English users, we have not found a good channel yet. If anyone can provide help, please leave your suggestions. Thank you all for your attention to the project.

**Comment by BowieHsu** [comment]
大家好，同步一下最新的进展：1.我们已经上传了yolov5l/yolov5m/yolov5s/yolov7的coco预训练模型，放置在1.0的release文件中了，可以直接用detect和val代码进行渲染和计算准确率，欢迎试用和提出相应问题；2.我们发布了做这项工作的背景介绍，方便您了解我们作为工业界的研发团队对半监督目标检测领域的认知：https://zhuanlan.zhihu.com/p/610959012

Hello everyone, here's an update: 1. We have uploaded pre-trained YOLOv5l/YOLOv5m/YOLOv5s/YOLOv7 models on COCO dataset. You can find these models in the 1.0 release files and directly use detect and val codes to render and compute accuracy. Feel free to try and ask any relevant questions. 2. We have also released a background introduction to our work, which helps you understand our R&D team's cognition on the semi-supervised object detection field in the industry: https://zhuanlan.zhihu.com/p/610959012.

**Comment by BowieHsu** [comment]
最新进展：我们上传了yolov7s-simota的训练脚本和coco预训练模型，方便各位用户通过比较yolov7.cfg和yolov7s-simota.cfg来了解如何将一个anchor based的检测器变成anchor free的检测器，这里面只会修改一点点配置文件，欢迎大家试用。

Latest update: We have uploaded the training script and coco pretrain model. Users can now compare yolov7.cfg and yolov7s-simota.cfg to learn how to convert an anchor-based detector to an anchor-free detector with just a few modifications to the configuration file. We encourage everyone to try it out.

**Comment by BowieHsu** [comment]
最新进展：我们上传了最新的COCO-additional实验结果，可以将标准的YOLOv5l从49.01的指标利用半监督提升到50.459，我们已经上传了最新训练出来的pt文件。

Latest update: We have uploaded the latest COCO-additional experimental results, which can boost the standard YOLOv5l from a score of 49.01 using semi-supervised learning to 50.459. We have also uploaded the latest trained pt file.

![截屏2023-03-08 上午8 45 50](https://user-images.githubusercontent.com/4092342/223589867-8dfc706f-0b54-46d7-acbb-cde2ebe4e5a0.png)
